test/manyvariables.jl

```diff
@@ -223,6 +225,8 @@ end
     @test eltype(convert(TaylorN{Complex{Float64}},1)) == TaylorN{Complex{Float64}}
     @test TS.numtype(convert(TaylorN{Complex{Float64}},1)) == Complex{Float64}
     @test normalize_taylor(xT) == xT
+    @test 1 > xT > yT > xT^2 > 0
```

Comment:
Maybe I'm misunderstanding something, but to me to it makes very little sense that `xT > yT` should hold  true. I would expect the Taylor series to be symmetrical with respect to which variable is put first.

Reply:
I agree with the expectation about the symmetry in labeling the variables; I expected that too... 

As I understand it, for many variables you have to impose something like that (which is the *lexicographic* part in the ordering), to have a *total* ordering (either `xT > yT` or `xT < yT`). Whatever lexicographic rule works fine. I've added some comments in the docstrings, hopefully making this clearer.